python/pyproject.toml

```diff
@@ -19,7 +19,7 @@ dependencies = ["requests", "tqdm", "numpy", "IPython", "setproctitle"]
 runtime_common = [
     "aiohttp", "decord", "fastapi",
     "hf_transfer", "huggingface_hub", "interegular", "modelscope",
-    "orjson", "outlines>=0.0.44,<0.1.0",
+    "orjson", "outlines>=0.0.44,<=0.1.11",
```

Comment:
The newer version isn't compatible with the latest main release.

Reply:
Seems fine to me, no error seen, have a specific?

Reply:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
vllm 0.6.4.post1 requires outlines<0.1,>=0.0.43, but you have outlines 0.1.11 which is incompatible.
Successfully installed outlines-0.1.11
```

NVIDIA GPU uses vllm 0.6.4.post1, with runtime_common as the shared component.

Reply:
solved with separation